src/screens/ConfigurePMTs/paymentMethodList.res

```diff
@@ -0,0 +1,112 @@
+@react.component
+let make = (~isPayoutFlow=false) => {
+  open PaymentMethodConfigUtils
+  open PaymentMethodEntity
+  let fetchConnectorListResponse = ConnectorListHook.useFetchConnectorList()
+  let businessProfiles = Recoil.useRecoilValueFromAtom(HyperswitchAtom.businessProfilesAtom)
+  let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Loading)
+  let (connectorResponse, setConnectorResponse) = React.useState(_ =>
+    Dict.make()->JSON.Encode.object
+  )
+  let filters = UrlUtils.useGetFilterDictFromUrl("")
+  let (filteredConnectors, setFiltersConnectors) = React.useState(_ =>
+    Dict.make()->JSON.Encode.object->getConnectedList
+  )
+  let (configuredConnectors, setConfiguredConnectors) = React.useState(_ =>
+    Dict.make()->JSON.Encode.object->getConnectedList
+  )
+  // let userPermissionJson = Recoil.useRecoilValueFromAtom(HyperswitchAtom.userPermissionAtom)
+  let (offset, setOffset) = React.useState(_ => 0)
+  let allFilters: PaymentMethodConfigTypes.paymentMethodConfigFilters = React.useMemo1(() => {
+    filters->pmtConfigFilter
+  }, [filters])
+  let getConnectorListAndUpdateState = React.useCallback0(async () => {
+    try {
+      setScreenState(_ => Loading)
+      let response = await fetchConnectorListResponse()
+      let configuredConnectors = response->getConnectedList
+      let filterdValue = response->getFilterdConnectorList(allFilters)
+      setFiltersConnectors(_ => filterdValue)
+      setConnectorResponse(_ => response)
+      setConfiguredConnectors(_ => configuredConnectors)
+      setScreenState(_ => Success)
+    } catch {
+    | _ => setScreenState(_ => PageLoaderWrapper.Error("Failed to fetch"))
+    }
+  })
+
+  React.useEffect1(() => {
+    RescriptReactRouter.replace(`/configure-pmts`)
+    getConnectorListAndUpdateState()->ignore
+    None
+  }, [isPayoutFlow])
+
+  let applyFilter = async () => {
+    setScreenState(_ => Loading)
+    let res = connectorResponse->getFilterdConnectorList(allFilters)
+    setFiltersConnectors(_ => res)
+    await HyperSwitchUtils.delay(500)
+    setScreenState(_ => Success)
+  }
+
+  React.useEffect1(() => {
+    let {connectorId, profileId, paymentMethod, paymentMethodType} = allFilters
+    if (
+      connectorId->Option.isSome ||
+      profileId->Option.isSome ||
+      paymentMethod->Option.isSome ||
+      paymentMethodType->Option.isSome
+    ) {
+      applyFilter()->ignore
+    }
+    None
+  }, [allFilters])
+
+  let handleClearFilter = async () => {
```

Comment:
Add try catch

Reply:
try catch is not required as its not going to make any api added async function because wanted to use the delay module